src/ORM/Filters/ExactMatchFilter.php

```diff
@@ -205,4 +221,38 @@ public function isEmpty()
     {
         return $this->getValue() === [] || $this->getValue() === null || $this->getValue() === '';
     }
+
+    /**
+     * Determine if we can use placeholders for the given column and values
+     * Current rules are to use placeholders for all values, unless:
+     * - The column is a DBPrimaryKey or DBForeignKey
+     * - The values being filtered are all either integers or valid integer strings
+     * - Using placeholders for integer ids has been configured off, which is the default config value
+     *
+     * Disabling placeholders large `WHERE ID in (id1, id2, ...) style queries was measured to be significantly
+     * faster if placeholders were disabled when querying a large number of ID e.g. over 1000
+     */
+    private function usePlaceholders(string $column, array $values): bool
+    {
+        if ($this->config()->get('use_placeholders_for_integer_ids')
+            || !preg_match('#^"(.+)"."(.+)"$#', $column, $matches)) {
```

Comment:
The regex is for `$col = $matches[2];` below

Reply:
It's obviously doing more than just getting that value, since we're explicitly saying to use placeholders if the regex doesn't match. What scenarios will there be a column that doesn't match that pattern that we're 'protecting' against?
_does_ this PR handle filtering against "MyRelation.ID"?

Reply:
Have separated out and added explanation